contrib/google.golang.org/grpc/grpc_test.go

```diff
@@ -591,3 +591,63 @@ func TestAnalyticsSettings(t *testing.T) {
 		assertRate(t, mt, 0.23, WithAnalyticsRate(0.23))
 	})
 }
+
+func TestIgnoredMethods(t *testing.T) {
+	for name, c := range map[string]struct {
+		rpcType     string
+		ignorePaths []string
+		expNumSpan  int
+	}{
+		"unary not ignored": {
```

Comment:
In Go the common idiom is to use only one word for test names. Can you please change the tests as suggested? 

```
func TestIgnoredMethods(t *testing.T) {
    t.Run("unary", ...
    t.Run("stream", ...
}
```
We don't want to have the `rpcType`,  the switch cases etc. Too much complexity.